blog/docs/articles/workflow-as-function-flow.md

```diff
@@ -0,0 +1,329 @@
+
+### Deployment
+
+First of all, the heavy work that requires parsing workflow definition, slicing that into
+functions and events and packaging into deployable unit is done at build time.
+
+Automatiko project that implements this concept is a java based implementation that takes advantage of [GraalVM](https://www.graalvm.org) to be able to compile into native executable that brings small memory footprint at runtime and lightning fast startup time.
+
+With that said, workflow as a function flow with Automatiko compiles into single service and by that into a single container image, that includes all the functions. Each and every function can be invoked at any time on any replica of the service.
+
+At the same time (during build) Knative manifest file is generated to ease deployment to the cluster. That consists of
+
+- Knative serving service
+- sink binding to inject broker location
+- triggers for each function created out of the workflow
+
+So this serves as a starting point to have fully runnable service directly after build. It can be modified further to adopt to particular configuration of Knative in the cluster.
+
+Enough theory, let's look at example of this concept.
+
+## User registration example
+
+This example it is a simple user registration that performs various checks and registers user in the Swagger PetStore service.
+
+See complete description of this example [here](https://docs.automatiko.io/main/0.0.0/examples/userregistration.html)
+
+This example comes with two flavours (depending of what DSL is used to create workflow definition)
+
+![Slice workflow into functions](/blog/articles/images/workflow-function-flow-function-example-bpmn.png)
+*BPMN*
+
+![Slice workflow into functions](/blog/articles/images/workflow-function-flow-function-example-sw.png)
+*Serverless Workflow Spec*
+
+To try out the example yourself clone one of the flavours of this example project
+
+* [User Registration BPMN](https://github.com/automatiko-io/automatiko-examples/tree/main/user-registration)
+* [User Registration Serverless Workflow](https://github.com/automatiko-io/automatiko-examples/tree/main/user-registration-sw)
+* [User Registration BPMN on Google Cloud Run](https://github.com/automatiko-io/automatiko-examples/tree/main/user-registration-gcp-cloudrun)
+
+and build the application by issuing following maven command from within the cloned repository
+
+````
+mvn clean package -Pcontainer-native
+````
+
+This will build a container with the service that is using native executable built with GraalVM so the build process will take a while but it will be really light and fast at execution.
+
+### Push built container to registry
+
+Once the container image is built push it to external registry that your Knative cluster can pull from.
+
+### Deploy to Knative cluster
+
+Create a Knative eventing broker, for example by using following command:
```

Comment:
These should be numbered steps for a procedure, FYI

Reply:
changed to numbered steps